BizTechApp/App/reducers/loginReducers.js

```diff
@@ -25,6 +25,10 @@ export default function loginReducer(state=initialState, action) {
                 isLoading: false,
                 error: action.err
             })
+        case 'logout':
+            return Object.assign({}, state, {
+              isLoggedIn: false
```

Comment:
I feel like logging out should also reset the user data to null, since logging in again will result in overriding the user field anyways, no?

Reply:
I fixed this, it's in one of the other PRs

Reply:
ohhh